CMakeLists.txt

```diff
@@ -529,6 +535,34 @@ else()
     CONFIGURE_FILE("${CMAKE_CURRENT_SOURCE_DIR}/src/lib/libplctag.h" "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/libplctag.h" COPYONLY)
 endif(ANDROID_BUILD)
 
+# add the cli
```

Comment:
Hmm, now that I am really looking into this.  I wonder if it would make more sense to break the entire CLI code out and make it have its own CMake file?  Then you could have a much simpler build process.